### ISSUE TITLE
FBSDKCrashHandler overwrite the old signal handler of my crash report framework

### ISSUE THREAD

**Issue by yonqiSong** [facebook/facebook-ios-sdk#1533]
if my signal handler register before this code, the handler will be overwrite and will never receive the signal again.

```
+ (void)installSignalHandler
{
  [self printSigHandler];
  signal(SIGBUS, FBSDKSignalHandler);
  signal(SIGFPE, FBSDKSignalHandler);
  signal(SIGILL, FBSDKSignalHandler);
  signal(SIGPIPE, FBSDKSignalHandler);
  signal(SIGSEGV, FBSDKSignalHandler);
  signal(SIGSYS, FBSDKSignalHandler);
}

```
the reason seems that FB signal handler doesn't send the signal to the old one.

```
void FBSDKSignalHandler(int sig)
{
  NSMutableArray<NSString *> *callStack = [[NSThread callStackSymbols] mutableCopy];
  if (callStack) {
    NSIndexSet *indexSet = [NSIndexSet indexSetWithIndexesInRange:NSMakeRange(0, 2)];
    if (callStack.count > 2 && [callStack objectsAtIndexes:indexSet]) {
      [callStack removeObjectsAtIndexes:indexSet];
    }
  }
  [FBSDKCrashHandler saveSignal:sig withCallStack:callStack];

  // reset to default handler
  signal(sig, SIG_DFL);
  // re-signal to default handler
  raise(sig);
}

```

**Comment by ZhihaoJun** [comment]
Same here.

**Comment by jaderYang** [comment]
Same here. 
It is recommended to provide a public function for developers to disable it.

**Comment by tianqibt** [comment]
We have released v8.2.0. We removed signal handler in this version. Please upgrade your SDK. Thanks!
